src/Common/src/Interop/Windows/user32/Interop.GetWindowLong.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Just curious, I see you're following the existing pattern, but why do we specify each pinvoke in its own file?